modules/landing-page/landing-footer.tsx

```
@@ -0,0 +1,76 @@
+            {
+              title: 'Resources',
+              links: ['Help Center', 'Community', 'Tutorials', 'Status', 'Changelog'],
+            },
+          ].map((column, index) => (
+            <div key={index}>
+              <h4 className="mb-6 font-bold text-black">{column.title}</h4>
+              <ul className="space-y-4">
+                {column.links.map((link, linkIndex) => (
+                  <li key={linkIndex}>
+                    <a
+                      href="#"
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)